src/test/java/com/atlassian/bitbucket/jenkins/internal/client/BitbucketRequestExecutorTest.java

```diff
@@ -0,0 +1,27 @@
+package com.atlassian.bitbucket.jenkins.internal.client;
+
+import com.atlassian.bitbucket.jenkins.internal.client.exception.NoContentException;
+import com.atlassian.bitbucket.jenkins.internal.fixture.FakeRemoteHttpServer;
+import com.atlassian.bitbucket.jenkins.internal.http.HttpRequestExecutorImpl;
+import org.junit.Test;
+
+import static com.atlassian.bitbucket.jenkins.internal.client.BitbucketCredentials.ANONYMOUS_CREDENTIALS;
+import static com.atlassian.bitbucket.jenkins.internal.util.TestUtils.BITBUCKET_BASE_URL;
+import static com.atlassian.bitbucket.jenkins.internal.util.TestUtils.OBJECT_MAPPER;
+
+public class BitbucketRequestExecutorTest {
```

Comment:
I think we should add unit tests for all the methods of `BitbucketRequestExecutor`

Reply:
Its been unit tested through `BitbucketClientFactoryImplTest`